src/components/link/index.tsx

```diff
@@ -53,18 +56,20 @@ const Link = forwardRef<HTMLAnchorElement, Props>(
     };
 
     return (
-      <Component
-        target={target || (isExternal ? '_blank' : undefined)}
-        rel={rel || (isExternal ? 'noopener noreferrer' : undefined)}
-        ref={ref}
-        aria-label={ariaLabel}
-        {...rest}
-        fontWeight={fontWeight}
-      >
-        {leftIcon}
-        <chakra.span __css={textStyle}>{children}</chakra.span>
-        {rightIcon}
-      </Component>
+      <Box>
```

Comment:
The link takes up the entire width of the page because it behaves as a block-level element. That means that it starts on a new line and take up the full width. We can change this behaviour by adding a wrapper.

Reply:
could that maybe be solved with the simple `inline-block` instead of a wrapper

Reply:
I will check in the link theme itself. And test it. Thank you 🌸

Reply:
you could try setting `width="max-content"` inside listings-web on the link it affects. Then no wrapper at all is needed

Reply:
true, true!
I will close the PRs and adjust the links in listings. Thanks you!